2406/1/koans/about_dice_project.rb

```diff
@@ -0,0 +1,87 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+# This class smells of :reek:InstanceVariableAssumption
+class DiceSet
+  attr_reader :values
+  def roll(value)
```

Comment:
it's to complex!!! just initialize Array and pass block!

Reply:
Fix it.